src/core/background/storage/options.js

```diff
@@ -47,6 +48,12 @@ define((require) => {
 	 */
 	DEFAULT_OPTIONS[USE_UNRECOGNIZED_SONG_NOTIFICATIONS] = false;
 
+	/**
+	 * Scrobble podcast episodes. Inherits the GoogleMusic setting.
+	 * @type {Boolean}
+	 */
+	DEFAULT_OPTIONS[SCROBBLE_PODCASTS] = undefined;
```

Comment:
Why this option is `undefined` by default?

If you want to force migration, this could be done better:
1. Check if the Google Music option is not `undefined`.
2. If true, assign the Google Music  option value to the new option; do nothing otherwise.
3. Remove Google Music option from storage.